### DISCUSSION TITLE
how to disable auto bedlevel every time I start a print over lan?

### DISCUSSION THREAD

**Discussion by dechaar82** [OrcaSlicer/OrcaSlicer#5570] [Q&A]
Hello, 
I updated to orcaslicer 2.1.0 and now the auto bedlevel starts if i send a print to my Ender V3 KE.
What do i need to remove in the gcode machine start tab ? I don't want it to calibrate all the time.
this one? : BED_MESH_CALIBRATE PROBE_COUNT=6,6 ;Auto bed level ?

![Schermafbeelding 2024-06-03 192601](https://github.com/SoftFever/OrcaSlicer/assets/162752526/824e3feb-e7de-48d5-bcf3-8c6e099c8ee2)
Thanks for the help 
greetings

**Comment by darco415** [comment]
well since we cant see your entire start gcode. looks for "G29 ; Auto bed leveling"

**Reply by dechaar82** [reply]
Thanks for the reply, I removed that piece of code and it is not calibrating anymore so it worked.

**Comment by dechaar82** [comment]
Thanks for the reply, I removed that piece of code and it is not calibrating anymore so it worked.

**Reply by lappa1** [reply]
I have exactly the same problem of auto bed level checks all the time since updating to V2.1. 
There is no G29 in the Gcode, only the new code line "BED_MESH_CALIBRATE PROBE_COUNT=6,6": that wasn't there in the previous version.
If you put a G29 code in after the G28, it will auto level check twice.
Even with the line "BED_MESH_CALIBRATE PROBE_COUNT=6,6": commented out, it still auto bed level checks -why?
Why was this line "BED_MESH_CALIBRATE PROBE_COUNT=6,6" added to V2.1 and 2.1.1?

**Reply by victorpigmeo** [reply]
I'm having this issue and there is no G29 code, it is calibrating 5,5 and then 6,6 just after, when I comment the BED_MESH_CALIBRATE PROBE_COUNT=6,6 it calibrates 5,5 still

**Comment by gunrunnerjohn** [comment]
I have a K1C and I'm running Orca 2.2.0.  I also have it calibrating before each print, and there is nothing in the initialization code that commands this action, so how do I convince Orca to stop calibrating every print?

![___START_G-CODE](https://github.com/user-attachments/assets/690ce442-6d88-48cf-8248-e61f8e7a65b3)
